Nodejs/Product/Nodejs/Project/NodejsProjectLauncher.cs

```diff
@@ -468,6 +464,29 @@ private bool ShouldStartBrowser()
             return true;
         }
 
+        private string GetBrowserPath()
+        {
+            ThreadHelper.ThrowIfNotOnUIThread(nameof(GetBrowserPath));
+
+            var uiShellOpenDocument = (IVsUIShellOpenDocument)ServiceProvider.GlobalProvider.GetService(typeof(SVsUIShellOpenDocument));
+            uiShellOpenDocument.GetFirstDefaultPreviewer(out var defaultBrowserPath, out _, out _);
+
+            return defaultBrowserPath;
+        }
+
+        private static void OpenBrowser(string browserPath, string webBrowserUrl)
+        {
+            // Chrome has known issues with being launched as admin.
+            if (!string.IsNullOrEmpty(browserPath) && browserPath.EndsWith(NodejsConstants.ChromeApplicationName))
+            {
+                SystemUtility.ExecuteProcessUnElevated(browserPath, webBrowserUrl);
```

Comment:
Any reason not to launch all browsers unelevated?

Reply:
Since I just tested this for Chrome, I decided it's a good idea to change only the behavior we know is broken. In any case, if you feel like we should always launch un-elevated I can do that.